README.md

```diff
@@ -92,6 +92,18 @@ Default: `''`
 
 Add an additional class to the input.
 
+#### suggestListClassName
```

Comment:
Can you name this `suggestsClassName` to be in line with the `style.suggests` and the `suggestsHiddenClassName` properties?